src/clj/villagebook/config.clj

```diff
@@ -1,7 +1,7 @@
 (ns villagebook.config
   (:require [villagebook.auth.backend :as backend]))
 
-(def db-spec "jdbc:postgresql://localhost:5432/villagebook?user=postgres")
+(def ^:dynamic db-spec "jdbc:postgresql://localhost:5432/villagebook")
```

Comment:
I was going to say move this to a config files, you have already done that in the other PR. So no comments :P